README.md

```diff
@@ -183,10 +183,32 @@ With reference to the [GitHub Commit status API], the `POST` parameters (`state`
 - `repo`\
   The GitHub repository name.
 
+
+###  Authentication keys
+
+Keys must be provided for one of the following authentication methods:
+1. [Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)
+1. [GitHub app](https://docs.github.com/en/apps/creating-github-apps/about-creating-github-apps/about-creating-github-apps)
+
+#### Personal Access Token keys
+
 - `access_token`\
   The OAuth access token.\
   See also: section [GitHub OAuth token](#github-oauth-token).
 
+#### GitHub app keys
+
+- `private_key` \
+  The private key generated for the GitHub application
+
+- `application_id`
+  The application id of the GitHub application
+
+- `installation_id`
```

Comment:
How do you find the installation id? The only way I managed to achieve this is with my custom script that:
- creates a signed JWT token
- do a GET request using the JWT to: `/app/installations` endpoint

The docs here:
https://github.com/bradleyfalzon/ghinstallation?tab=readme-ov-file#what-is-app-id-and-installation-id

is wrong...

> Settings > Developer > settings > GitHub Apps > Advanced > Payload in Request tab

doesn't exists.. at least not for me with my test app and webhooks inactive... (which is what you are suggesting a few lines above..)

Reply:
If you go to 

https://github.com/settings/apps/test-cogito/installations
and then click the gear icon  its in the URL
https://github.com/settings/installations/<ID>

Reply:
I don't have access to these pages... Can you post a screenshot?

Just curious:
- is this organization app or your user app?
- are webhooks active on it or not?

When I visit those pages for my test app, there is no mention of installation ids... only client id and app id...

Reply:
i tested with both and organization app and user app.   webhooks are not required.    installation id is in the URL that the configure button takes you too

Reply:
i..e https://github.com/settings/installations/63671192 the ID is the installation id

Reply:
Yes... it really is :) it was not evident for me, I must say. Thanks for clarifying this..

We will need to update the docs explaining in detail (even with screenshot) if necessary how to find this value. We should probably also offer the alternative way with a one-off script...